src/main/Save.tsx

```diff
@@ -110,7 +111,7 @@ export const SaveButton: React.FC<{}> = () => {
   // Update based on current fetching status
   let icon = faSave
   let spin = false
-  let tooltip = t("save.confirmButton-default-tooltip")
+  let tooltip = ""
```

Comment:
Why set the tooltip to the empty string? What does this accomplish?

Reply:
Mainly two reasons:

- The first reason: the initial tooltip for the save-button would be "Save Button" and would be the only tooltip for this kind of buttons, this would not be consistent. When it`s empty, there is no tooltip shown. So it's partly a design decision to 'remove' this tooltip (also a tooltip should contain additional information to the user not repeating the elements text)
- The second reason for the empty string is, we need the variable and the title-attribute for the tooltip-wrapper has to be a string.

I will set the variable to null and then check for it in the tooltip-wrapper. If it's still null I will set it to an empty string

Reply:
> the initial tooltip for the save-button would be "Save Button" and would be the only tooltip for this kind of buttons, this would not be consistent. 

Makes sense, I can get behind that.

> The second reason for the empty string is, we need the variable and the title-attribute for the tooltip-wrapper has to be a string.

In all other cases, if a tooltip should not be displayed, the tooltip-wrapper is simply omitted (which is preferred I think, less visual clutter). Why do it differently in this case?

Reply:
> In all other cases, if a tooltip should not be displayed, the tooltip-wrapper is simply omitted (which is preferred I think, less visual clutter). Why do it differently in this case?

The button has multiple tooltips (for failing, loading and success), based on the workflowstatus. (I'm not sure why).
In my opinion we can also remove them. Then we don't need the tooltip-wrapper for this class.

Reply:
Aaah right, forgot about that. Maybe you could check if that is somehow relevant for screenreader folks. If not, all the tooltips can go. Makes no sense to remove one and keep the others.